tests/builtins/test_ascii.py

```diff
@@ -2,28 +2,45 @@
 
 
 class AsciiTests(TranspileTestCase):
-    pass
+    def test_ascii_without_unicode(self):
+        self.assertCodeExecution("""
+            print(ascii("Hello World!"))
+            print(ascii('\u0041'))
+
+            string = ''
+            for i in range(100):
+                print(ascii(chr(i)))
+                string += chr(i)
+            print(ascii(string))
+            """)
+
+    def tset_ascii_list(self):
```

Comment:
This test might not be run right now, the method name should start with `test_`.